apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -165,6 +171,7 @@ class PageTemplateModal extends Component {
 				className="page-template-modal"
 				overlayClassName="page-template-modal-screen-overlay"
 				shouldCloseOnClickOutside={ false }
+				isDismissable={ false }
```

Comment:
Perhaps a comment here explaining this would be helpful? It would be very easy to see this as an error rather than an intentional fix 😄

Reply:
added a comment